player_darwin.h

```diff
@@ -0,0 +1,137 @@
+#include <AudioToolbox/AudioToolbox.h>
+
+//--------------------------------------------
+// AudioPlayer Struct and Associated Functions
+//--------------------------------------------
+
+/*!
+ * AudioPlayer
+ * @field inputUnit - The unit to use to input Audio to the buffer
+ * @field outputUnit - The unit to use to output Audio to the HAL
+ */
```

Comment:
I understand this is the Apple standard, but Oto is not software only for Apple. The problem is that Oto doesn't have a strong style rule, but other code in Oto should use only `//`. Let's use `//` just for consistency.